src/dynamic-side-content.scss

```diff
@@ -0,0 +1,101 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-dynamic-side;
+
+.#{$block} {
+  $main: #{$block}__main;
+  $side: #{$block}__side;
+
+  $main-width-screen-s: 100%;
```

Comment:
Can we name the inflection names with 2 letters (`sm`, `md`)

Reply:
Done